custom/conf/app.example.ini

```diff
@@ -907,6 +907,24 @@ LEVEL = Info
 ;; Valid site url schemes for user profiles
 ;VALID_SITE_URL_SCHEMES=http,https
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;[service.explore]
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; Only allow signed in users to view the explore pages.
+;REQUIRE_SIGNIN_VIEW = false
```

Comment:
what is the difference with `service.REQUIRE_SIGNIN_VIEW`?

Reply:
> A global setting and a special setting?

Yes. `service.REQUIRE_SIGNIN_VIEW` is global. `service.explore.REQUIRE_SIGNIN_VIEW` is only for explore pages.

`service.explore.REQUIRE_SIGNIN_VIEW`  works when the user does not want to show repo list and user list to public.

Reply:
![image](https://github.com/user-attachments/assets/a7f9ff43-9970-4d15-bcab-b52de8e805d5)
Then it seems that this function is not correct, it only checked `service.explore.REQUIRE_SIGNIN_VIEW`.
And in web side, it is right:
![image](https://github.com/user-attachments/assets/6598314e-ef10-480c-a41e-b9d6fd775e81)
![image](https://github.com/user-attachments/assets/10cdad81-3517-4d0c-8dae-88b7c903dab4)

Reply:
03292a8912a87bb9eab1229fa44b284d2f22acce